contrib/convert_image_to_build.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convert a docker save JSON files to a BUILD file so docker_import can read it."""
+
+import gzip
+import json
+import os
+import os.path
+import shutil
+import sys
+import tarfile
+
+def convert_manifest(infile, outfile):
```

Comment:
I think docker builds are not inherently reproducible, each time you do a docker build (with a clean cache), it will insert some timestamps into the manifest/config. It might be worth stripping those out here.

Reply:
This should go away completely in favor of `container_load`.  This is an interesting twist for the `WORKSPACE` discussion: how do you "lock" this kind of rule? :)